assets/js/pages/SettingsPage/SettingsPage.jsx

```diff
@@ -119,10 +134,42 @@ function SettingsPage() {
                     )}
                   </code>
                 </div>
+                {apiKey && (
+                  <div className="flex space-x-2 my-4">
+                    <EOS_INFO_OUTLINED size="20" className="mt-2" />
+
+                    <div className="mt-2 text-gray-600 text-sm">
+                      {apiKeyExpiration
+                        ? `Key Will Expire ${format(
+                            parseISO(apiKeyExpiration),
+                            'd LLL yyyy'
+                          )}`
+                        : 'Key will never expire'}
+                    </div>
+                    <div className="!ml-auto">
+                      <Button
+                        onClick={() => {
+                          window.navigator.clipboard.writeText(apiKey);
+                        }}
+                        type="primary-white"
+                      >
+                        Copy Key
```

Comment:
Let's use the same `copy` icon here, as we do in the modal.
The same thing in both at least. What do you think @jagabomb ?